### PR TITLE
feat(worker): pass platform options to onboarding pr

### LINKED ISSUE #7613: PlatformOptions are not passed to onboardingPR
**What Renovate type, platform and version are you using?**

any self-hosted until now

**Describe the bug**

Renovate doesn't pass `platformOptions` to onboarding pr. see renovatebot/config-help#955

**Relevant debug logs**
not relevant

**To Reproduce**

**Additional context**

On normal pr, `platformOptions` are passed to `createPr`
https://github.com/renovatebot/renovate/blob/c57bb612940831e1b6be29c1a86affe05bb92fee/lib/workers/pr/index.ts#L373-L381

But not at onboarding:
https://github.com/renovatebot/renovate/blob/0ea1413e57a482c502f8a90226e063c636669f4f/lib/workers/repository/onboarding/pr/index.ts#L134-L140

### PR DESCRIPTION
**Pull request by viceice**
## Changes:

We now pass the platform pr options to the onboarding pr.

## Context:

closes #7613

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

**Comment by viceice** [comment]
@JamieMagee @rarkins Do you see any risks here?

**Comment by rarkins** [comment]
There seems to be a small risk that we accidentally trigger GitLab's automerge on green feature. Even if it doesn't happen now, a refactoring in future would potentially trigger it?

**Comment by viceice** [comment]
> There seems to be a small risk that we accidentally trigger GitLab's automerge on green feature. Even if it doesn't happen now, a refactoring in future would potentially trigger it?

Uhh, yes i think it can happen. Do we set `automerge=false` explicit for onboarding pr?

**Review by JamieMagee — DISMISSED** [review]
Code changes look fine. Just the outstanding question around automerge of onboarding PRs for hosted Renovate

**Review by rarkins — CHANGES_REQUESTED** [review]
Disable autoMerge for onboarding

**Comment by viceice** [comment]
> Disable autoMerge for onboarding

any hint where to add this flag? 🤔

**Comment by viceice** [comment]
maybe here?
https://github.com/renovatebot/renovate/blob/80b691c807a6e2b6c98bcaab8198bf8932fcfa4e/lib/workers/repository/onboarding/branch/index.ts#L61

File: lib/workers/pr/index.ts [hunk 1]
```diff
@@ -98,6 +98,20 @@ export async function addAssigneesReviewers(
   }
 }
 
+export function getPlafromPrOptions(
```
Comment (rarkins) [inline lib/workers/pr/index.ts:101]:
```suggestion
export function getPlatfromPrOptions(
```

File: lib/workers/pr/index.ts [hunk 2]
```diff
@@ -376,7 +381,7 @@ export async function ensurePr(
           prTitle,
           prBody,
           labels: config.labels,
-          platformOptions,
+          platformOptions: getPlafromPrOptions(config),
```
Comment (rarkins) [inline lib/workers/pr/index.ts:384]: Maybe just mutate this result here, e.g. overriding `gitlabAutomerge` to set it to false?

**Review by rarkins — CHANGES_REQUESTED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**